src/lib/zod/index.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+
+export const createCircleSchemaInput = z
+  .object({
+    user_name: z.string().min(3).max(255),
+    address: z.string().regex(/^(0x)?[0-9a-f]{40}$/i),
```

Comment:
we have a address checker in `src/forms/formHelpers.ts`

Reply:
tried importing, backend seem to have no access to config/env

Cannot find module 'config/env'
Require stack:
- /yearn-coordinape/src/forms/formHelpers.ts
- /yearn-coordinape/src/lib/zod/index.ts
- /yearn-coordinape/api/hasura/actions/create_circle.ts
- /.config/yarn/global/node_modules/@vercel/node/dist/launcher.js
- /.config/yarn/global/node_modules/@vercel/node/dist/dev-server.js

Reply:
try using full relative paths (i.e. '../config/env' or whatever it should be)
it's only Webpack that makes "magic" paths like 'config/env' work, and ofc it doesn't run in the backend